openpype/modules/deadline/plugins/publish/submit_publish_job.py

```diff
@@ -126,22 +126,19 @@ class ProcessSubmittedJobOnFarm(pyblish.api.InstancePlugin):
                   "harmony": [r".*"],  # for everything from AE
                   "celaction": [r".*"]}
 
-    enviro_filter = [
+    environ_job_filter = [
+        "OPENPYPE_METADATA_FILE",
+        "OPENPYPE_PUBLISH_JOB",
+        "OPENPYPE_RENDER_JOB",
+        "OPENPYPE_LOG_NO_COLORS"
```

Comment:
I think these should be set explicitly by this plugin and should not get propagated from global os.
```suggestion
```